api-services/helpers.js

```diff
@@ -32,7 +32,7 @@ export function addMaxBodyLengthToRequestConfiguration (requestConfiguration, ma
     return requestConfiguration
 }
 
-export function buildApiRequestConfig (url, headers, data) {
+export function buildApiRequestConfig (url, headers, data, loggingProperties) {
```

Comment:
Might be worth having `loggingProperties = null` so devs know that it's an optional param (ie won't break anything if absent)

Reply:
Updating to make optionals null